### PR TITLE
Update Python dependencies to resolve security issues.

### PR DESCRIPTION
**Pull request by seriva**
- Updated first line dependency azure-cli from 2.8.0 to 2.23.0  solving the following security issues:
  1: jinja2 from 2.11.2 to 2.11.3
  2: urllib3 from 1.26.2 to 1.26.4
  3: cryptography from 2.9.2 to 3.3.2
- Updated all 3 line dependencies MINOR.PATCH versions if possible
- Updated of the licenses.py
- Updated of the COMPONENTS.md

### CODE REVIEW

File: core/src/epicli/gen-dependency-info.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def _main() -> None:
     # Write licenses 'cli/licenses.py'
     licenses_content = """\
     # This is a generated file so don`t change this manually. 
-    # To re-generate run 'python gen-licenses.py' from the project root.
+    # To re-generate run 'python gen-dependency-info.py 'GitHub PAC' ' from the project root.
```
Comment (to-bar) [inline core/src/epicli/gen-dependency-info.py]:
```suggestion
    # To re-generate run 'python gen-dependency-info.py <GitHub_PAT>' from the project root.
```

File: core/src/epicli/cli/licenses.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # This is a generated file so don`t change this manually. 
-# To re-generate run 'python gen-licenses.py' from the project root.
+# To re-generate run 'python gen-dependency-info.py 'GitHub PAC' ' from the project root.
```
Comment (to-bar) [inline core/src/epicli/cli/licenses.py]:
```suggestion
# To re-generate run 'python gen-dependency-info.py <GitHub_PAT>' from the project root.
```

**Review by to-bar — CHANGES_REQUESTED** [review]

**Comment by to-bar** [comment]
How about updating changelog?

**Comment by seriva** [comment]
> How about updating changelog?

Added.

**Review by plirglo — APPROVED** [review]

**Review by atsikham — APPROVED** [review]
LGTM

**Review by to-bar — APPROVED** [review]

**Pull request merged**